shared/tracker/index.js

```diff
@@ -131,6 +131,7 @@ export default connect(
   },
   (stateProps, dispatchProps, ownProps) => {
     return {
+      proofs: [],
```

Comment:
How can I repro? I tried a couple trackers and they all seem to have this state.